### PR TITLE
add paddleflow cli

### PR DESCRIPTION
**Pull request by qiaoshuangshuang**
add paddleflow cli

### CODE REVIEW

File: client/setup.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+with open('requirements.txt') as f:
+    REQUIRES = f.readlines()
+
+
+setup(
+    name=NAME,
+    version=VERSION,
+    description='PaddleFlow SDK',
+    author='The PaddleFlow Authors',
+    url="https://console.cloud.baidu-int.com/devops/icode/repos/baidu/bmlc/paddleflow/tree/master:client",
```
Comment (D0m021ng) [inline client/setup.py]: Internal sensitive information.

File: Makefile [hunk 2]
```diff
@@ -35,15 +35,15 @@ prepare: gomod
 
 gomod:
 	$(GO) env -w GO111MODULE=on
-	$(GO) env -w GOPROXY=https://goproxy.io,direct
-	$(GO) env -w CGO_ENABLED=0
+	$(GO) env -w GOPROXY=https://goproxy.cn,direct
+	$(GO) env -w CGO_ENABLED=1
```
Comment (D0m021ng) [inline Makefile]: CGO_ENABLED=1 causes dynamic link, and compiled binary cannot run on Alpine OS.

**Review by D0m021ng — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (qiaoshuangshuang) [reply Makefile]: update done

File: client/setup.py (hunk 1 above)
Comment (qiaoshuangshuang) [reply client/setup.py]: update to github

**Pull request merged**